tests/conftest.py

```diff
@@ -22,6 +29,38 @@ def clean_global_registry():
         yield None
 
 
+class ReusingWSGIServer(WSGIServer):
+    """A simple WSGI server which allows reusing the port.
+
+    There is of course a good reason for the existence of TIME_WAIT,
```

Comment:
TIME_WAIT is a state that a listening TCP port is put into after it is closed, preventing another application from reusing that port immediately. The reason for this is that clients may still be trying to connect to the now-shutdown server, and if the new server program uses the same port then it can intercept those connections, which is a potential security issue. However, since we don't want to wait a couple of minutes between tests, and there's no actual sensitive anything in there, I'm disabling this feature by setting option.

I'm not sure that this whole explanation should be in the code there though...

Reply:
OK, thanks for the clarification! Someone who doesn't know anything about this (like me) will not know that the comment as it is now actually refers to the allow_reuse_address option. So maybe make that a bit more explicit?

Reply:
Yeah, I wrote that comment with the intended users in mind, who are computer scientists in the field of network engineering...they know what I mean. I've reworded it, maybe it works better now for you as well?

Reply:
Alright. After your rewording I still don't fully understand it (you still have to know what TIME_WAIT is), but at least it is clear in what direction I would need to dig up some context if I want to edit this code.